src/packages/admin-ui-components/src/utils/export-csv.ts

```diff
@@ -0,0 +1,47 @@
+import Papa from 'papaparse';
+import type { Column } from 'react-data-grid';
+import { SelectOption } from '../multi-select';
+
+export const getExportDataFromSource = <T>(columns: Column<T, unknown>[], source: T[]) => {
+	const data =
+		source.map((row, index) =>
+			columns.map((c) => {
+				const value = row[c.key as keyof T] as any;
```

Comment:
Can we type this as at least having a label of type string? Is that possible?

Reply:
As discussed with @steve-keep, all object values will be displayed as string in csv.
<img width="1644" alt="Screenshot 2023-12-19 at 3 30 19 pm" src="https://github.com/exogee-technology/graphweaver/assets/29393307/47c05be9-b615-49eb-b3b0-41572ea41854">